src/core/getters/params.ts

```diff
@@ -89,7 +103,7 @@ export const getParams = ({
       }${
         !resolvedValue.originalSchema!.default
           ? `: ${resolvedValue.value}`
-          : `= ${resolvedValue.originalSchema!.default}`
+          : `= ${objToString(resolvedValue.originalSchema!.default)}`
```

Comment:
I already have a util function for that you can find it [here](https://github.com/anymaniax/orval/blob/master/src/utils/string.ts#L11)

Reply:
Thanks, exchanged it